src/commandDetails/miscellaneous/office.ts

```diff
@@ -0,0 +1,49 @@
+import { EmbedBuilder } from 'discord.js';
+import {
+  CodeyCommandDetails,
+  SapphireMessageExecuteType,
+  SapphireMessageResponse,
+} from '../../codeyCommand';
+import { DEFAULT_EMBED_COLOUR } from '../../utils/embeds';
+
+const officeExecuteCommand: SapphireMessageExecuteType = async (
+  _client,
+  _messageFromUser,
+  _args,
+): Promise<SapphireMessageResponse> => {
+  const officeEmbed = new EmbedBuilder()
+    .setColor(DEFAULT_EMBED_COLOUR)
+    .setTitle(`About the CSC Office`)
+    .setThumbnail('https://cdn.discordapp.com/emojis/869377257586704407.png')
+    .setDescription(`Find us in MC 3036/3037!`)
+    .addFields([
+      {
+        name: `What we offer:`,
+        value: `
+                •  Pop for just 50 cents
+                      •  Informative books
+                      • 5 computer terminals
+                      • (sometimes) super knowledgeable people `,
+      },
+      {
+        name: `Call us!`,
+        value: `(519) 888-4567 x33870`,
+      },
+    ]);
+  // look at info.ts about coin to see embded details
+  return { embeds: [officeEmbed] };
+};
+
+export const officeCommandDetails: CodeyCommandDetails = {
+  name: 'office',
+  aliases: [],
+  description: 'Get information about the CSC office.',
+  detailedDescription: `no detailed description yet`,
```

Comment:
Preferably, we fill in detailedDescription so that our commandWiki is complete. See attached example from member.ts.
<img width="795" alt="Screenshot 2024-11-22 at 8 19 56 PM" src="https://github.com/user-attachments/assets/05b62fe5-31dd-4376-9945-1b602380f0b5">